components/hab/src/command/origin/key/download.rs

```diff
@@ -137,9 +137,10 @@ fn handle_encryption(
         return Ok(());
     }
 
-    ui.begin(
-        format!("Downloading public encryption origin key for {}", origin),
-    )?;
+    ui.begin(format!(
+        "Downloading public encryption origin key for {}",
```

Comment:
I'm guessing this is simply `rustfmt` reformatting, but now that I read this message, it doesn't really make sense: origin keys aren't about encrypting anything, they're all about signing and verifying things (namely packages). I wonder if taking out the word "encryption" in the origin keys messaging might help to distinguish these 2 concepts in the future.

Reply:
This function is handling the case specific to origin encryption keys (they were a new concept introduced in a previous PR)